README.md

```diff
@@ -1,70 +1,90 @@
+|\-\-expose-path|Metrics expose path|\-\-expose-path=/metrics_new|  
+|\-\-expose-port|HTTP expose server port|\-\-expose-port=9216|  
+|-D, --debug|Enable debug mode||  
+|--version|Show version and exit|  
+  
+### Running the exporter  
+It you built the exporter using the method mentioned in the previous section, the generated binaries are in `mnogo_exporter_linux_amd64/mnogo_exporter` or `mnogo_exporter_darwin_amd64/mnogo_exporter`  
+  
+#### Example: running the exporter connecting to sandbox's primary  
+```  
+mnogo_exporter_linux_amd64/mnogo_exporter --mongodbdsn=mongodb://127.0.0.1:17001  
+```  
+#### Enabling collstats metrics gathering  
+`--mongodb.collstats-colls` receives a list of databases and collections to monitor using collstats.  
+Usage example: `--mongodb.collstats-colls=database1.collection1,database2.collection2`
 
-### Build the exporter
-The build process uses the dockerized version of goreleaser so you don't need to install Go.
-Just run `make build` and the new binaries will be generated under the build directory.
+#### Enabling compatibility mode.
+When compatibility mode is enabled by the `--compatible-mode`, the exporter will expose all new metrics with the new naming and labeling schema and at the same time will expose metrics in the version 1 compatible way.   
+For example, if compatibility mode is enabled, the metric `mongodb_ss_wt_log_log_bytes_written` (new format) 
 ```
-├── build
-│   ├── config.yaml
-│   ├── mnogo_exporter_7c73946_checksums.txt
-│   ├── mnogo_exporter-7c73946.darwin-amd64.tar.gz
-│   ├── mnogo_exporter-7c73946.linux-amd64.tar.gz
-│   ├── mnogo_exporter_darwin_amd64
-│   │   └── mnogo_exporter   <--- MacOS binary
-│   └── mnogo_exporter_linux_amd64
-│       └── mnogo_exporter   <--- Linux binary
+# HELP mongodb_ss_wt_log_log_bytes_written serverStatus.wiredTiger.log.
+# TYPE mongodb_ss_wt_log_log_bytes_written untyped
+mongodb_ss_wt_log_log_bytes_written 2.6208e+06
 ```
-### Running the exporter
-It you built the exporter using the method mentioned in the previous section, the generated binaries are in `mnogo_exporter_linux_amd64/mnogo_exporter` or `mnogo_exporter_darwin_amd64/mnogo_exporter` 
-
-#### Example: running the exporter connecting to sandbox's primary
+will be also exposed as `mongodb_mongod_wiredtiger_log_bytes_total`  with the `unwritten` label.
 ```
-mnogo_exporter_linux_amd64/mnogo_exporter --mongodbdsn=mongodb://127.0.0.1:17001
+HELP mongodb_mongod_wiredtiger_log_bytes_total mongodb_mongod_wiredtiger_log_bytes_total
+# TYPE mongodb_mongod_wiredtiger_log_bytes_total untyped
+mongodb_mongod_wiredtiger_log_bytes_total{type="unwritten"} 2.6208e+06
 ```
-#### Enabling collstats metrics gathering
-`--mongodb.collstats-colls` receives a list of databases and collections to monitor using collstats. 
-Usage example: `--mongodb.collstats-colls=database1.collection1,database2.collection2`
 
+## Initializing the development environment  
```

Comment:
Github has Contributing.MD for the development, maybe it's better to move it there